static/img/flags/de.svg

```diff
@@ -0,0 +1,83 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
That seems to be a Go library for programatically creating SVGs. Is it possible that you meant https://github.com/svg/svgo (the Node.js optimizer for SVGs)?